src/d3a/models/strategy/__init__.py

```diff
@@ -588,6 +617,14 @@ def can_bid_be_posted(
 
         return total_posted_energy <= required_energy_kWh and bid_price >= 0.0
 
+    def can_settlement_bid_be_posted(self, bid_energy, bid_price, market, replace_existing=False):
```

Comment:
Can't find its usage, also of `can_settlement_offer_be_posted `

Reply:
Will be used in this ticket https://gridsingularity.atlassian.net/browse/D3ASIM-3567 , but was easier to be introduced by this one.